### PR TITLE
Catalog file for Vets Website for the Console UI.

### PR DESCRIPTION
**Pull request by jchapin**
## Description

The addition of this configuration file will allow the Console UI to give the vets-website a catalog entry and profile page in the Console UI. Additionally it will allow the Console UI to monitor and update all of the tools connected to the vets-website project.

## Original issue(s)

Related to:

department-of-veterans-affairs/platform-console-ui#338

and

department-of-veterans-affairs/platform-console-ui#275

Right now the Console UI has the platform-console-ui component and its dependencies recorded. Given that vets-website is a key dependency for the majority of VFS apps, we want to set it up in the Console UI as soon as practicable. It will also help test our auto-discovery of projects in the organization which should be referenced from the Console UI (338) and test our ability to identify projects which should be a part of the Console UI, but have not been onboarded yet.

## Testing done

We have onboarded vets-api with a similar configuration file:

https://github.com/department-of-veterans-affairs/vets-api/pull/9679

## Screenshots

Screen Shot 2022-04-20 at 5 32 53 PM

## Acceptance Criteria

This is simply a configuration file consumed by another system.

We have a similar, but more complex file in our own repository:

https://github.com/department-of-veterans-affairs/platform-console-ui/blob/main/catalog-info.yaml

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
```
Comment (jchapin) [inline catalog-info.yaml:2]: A Component describes a software component. It is typically intimately linked to the source code that constitutes the component, and should be what a developer may regard a "unit of software", usually with a distinct deployable or linkable artifact.

https://backstage.io/docs/features/software-catalog/descriptor-format#kind-component

File: catalog-info.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+metadata:
+  name: vets-website
```
Comment (jchapin) [inline catalog-info.yaml:4]: Short name for the component. Matching the repo-name.

File: catalog-info.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  description: This is the front end for VA.gov.
```
Comment (jchapin) [inline catalog-info.yaml:5]: Taken from the README. Dropping the "repository".

File: catalog-info.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  annotations:
+    github.com/project-slug: department-of-veterans-affairs/vets-website
```
Comment (jchapin) [inline catalog-info.yaml:7]: GitHub slug... owner/repo

https://github.com/department-of-veterans-affairs/vets-website

File: catalog-info.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  tags:
+    - javascript
+    - sass
```
Comment (jchapin) [inline catalog-info.yaml:10]: Two sample/example tags.

File: catalog-info.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+spec:
+  type: website
```
Comment (jchapin) [inline catalog-info.yaml:12]: free-form but:

```
The current set of well-known and common values for this field is:

service - a backend service, typically exposing an API
website - a website
library - a software library, such as an npm module or a Java library
```

File: catalog-info.yaml [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  owner: platform-release-tools
```
Comment (jchapin) [inline catalog-info.yaml:13]: GitHub team name for @department-of-veterans-affairs/platform-release-tools team.

File: catalog-info.yaml [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+  lifecycle: production
```
Comment (jchapin) [inline catalog-info.yaml:14]: Currently in production.

**Review by alexandec — APPROVED** [review]

**Pull request merged**